Pal 1090 services ReadMe updates (#358)
gchq/Palisade-services: topic-offset-service/README.md

(fields marked with * are acquired from headers metadata)

The Topic-Offset Service monitors and watches the `masked-resource` Kafka topic and reports the offsets that satisfy some prerequisites. 

> This service accepts a `TopicOffsetRequest`, which contains a token value, the service will then look at the value of the `x-request-token` header to check if it is a `START` message. 

Reviewer: ```suggestion
This service retrieves the values of the `x-request-token` and `x-stream-marker` headers to decide if the `START` has been found for this token. 
```

Reviewer: At the very least, the START/END comes from the `x-stream-marker` header, not the `x-request-token` header

Author: Updated
